src/main/java/frc/lib/team3061/vision/Vision.java

```diff
@@ -96,11 +97,15 @@ public void periodic() {
                     .getTranslation()
                     .getNorm()
                 < MAX_POSE_DIFFERENCE_METERS) {
-              poseEstimator.addVisionMeasurement(robotPose.toPose2d(), getLatestTimestamp());
+              if (vision_subsystem_is_enabled) {
+                poseEstimator.addVisionMeasurement(robotPose.toPose2d(), getLatestTimestamp());
+              }
 
               Logger.getInstance().recordOutput("Vision/TagPose", tagPose);
               Logger.getInstance().recordOutput("Vision/CameraPose", cameraPose);
               Logger.getInstance().recordOutput("Vision/RobotPose", robotPose.toPose2d());
+              Logger.getInstance()
+                  .recordOutput("Vision/vision_subsystem_is_enabled", vision_subsystem_is_enabled);
```

Comment:
For the tag (i.e., "vision_subsystem_is_enabled"), just use isEnabled to keep it brief. There isn't a lot of room the AdvantageScope interface for long names.

Reply:
done